test/evergreen.yml

```
@@ -147,7 +151,11 @@ functions:
             . automation-scripts/evergreen/find_gperftools.sh ${s3_access_key} ${s3_secret_key} ${build_variant} $is_cmake_build
           fi
           # Compiling with CMake.
-          . test/evergreen/find_cmake.sh
+          if [ "${cmake_binary|}" ]; then
+            CMAKE=${cmake_binary|}
```

Should we do instead:
```suggestion
          if [ -n "${cmake_binary}" ]; then
            CMAKE=${cmake_binary}
          fi 
```

---

removed the if statement since we don't need it anymore.